src/exchangeRateRouter.ts

```diff
@@ -71,6 +75,18 @@ const v1ExchangeRateIsoSubtractor = (
   next()
 }
 
+const v1IsoChecker = (req: any, res: any, next: Function): void => {
+  if (
+    req.requestedRates.data.every(pair => !isIsoPair(pair.currency_pair)) !==
+    true
```

Comment:
Too much negated logic. Did I get this correct: `if (req.requestedRates.data.some(pair => isIsoPair(pair.currency_pair)))`?